finalPrep/README.md

```diff
@@ -31,24 +31,24 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+ 
+<img width="1234" alt="Screen Shot 2021-09-10 at 6 25 14 PM" src="https://user-images.githubusercontent.com/78129211/132935020-f0988c54-39e9-4630-bf90-c51d544aac9b.png">
+- `Add Week 3 Screenshot Here` Honestly, I wouldn't change anything across weeks. At this point I don't know enough about the schedule to individualize the weeks. Currently, I'd keep everything the same week to week. 
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [My Mentorship DTR](https://gist.github.com/jasmyn2244/31f4adbbf19f9b6118e77e048887c037)
```

Comment:
Make sure you don't have typos in your note to your mentor! Treat this as a professional relationship.